sites/example-project/src/components/modules/formats.js

```diff
@@ -6,142 +6,222 @@ import { getContext } from "svelte";
 export const builtInFormats = [
   // Date/Time:
   {
-    formatName: "date",
-    formatValue: "MMM dd, yyyy",
+    formatTag: "date",
+    formatCode: "MMM d, yyyy",
     valueType: "date",
     exampleInput: "Jan 9 2022 07:32:04",
   },
   {
-    formatName: "month",
-    formatValue: "mmm",
+    formatTag: "month",
+    formatCode: "mmm",
     valueType: "date",
     exampleInput: "2022-01-09",
   },
   {
-    formatName: "year",
-    formatValue: "yyyy",
+    formatTag: "year",
+    formatCode: "yyyy",
     valueType: "date",
-    exampleInput: "2022/01/09"
+    exampleInput: "2022/01/09",
   },
 
   // Currency:
   {
-    formatName: "usd",
-    formatValue: "$#,##0.00",
+    formatTag: "usd",
+    formatCode: "$#,##0.00",
+    axisFormatCode: "$#,##0",
     valueType: "number",
     exampleInput: 101.1,
+    titleTagReplacement: " ($)",
   },
   {
-    formatName: "eur",
-    formatValue: "€#,##0.00",
+    formatTag: "eur",
+    formatCode: "€#,##0.00",
+    axisFormatCode: "€#,##0",
+    titleTagReplacement: " (€)",
     valueType: "number",
     exampleInput: 101.1,
   },
   {
-    formatName: "jpy",
-    formatValue: '"¥"#,##0.00',
+    formatTag: "jpy",
+    formatCode: '"¥"#,##0.00',
+    axisFormatCode: '"¥"#,##0',
+    titleTagReplacement: " (¥)",
     valueType: "number",
     exampleInput: 101.1,
   },
   {
-    formatName: "gbp",
-    formatValue: '"£"#,##0.00',
```

Comment:
Added ability to have a separate formatting for axis labels and titles (not exposed for custom formats). In theory, these could eventually be functions depending on where we want to take them.